rvic/core/read_forcing.py

```diff
@@ -19,6 +21,19 @@
 log = getLogger(LOG_NAME)
 # -------------------------------------------------------------------- #
 
+# -------------------------------------------------------------------- #
+# create Listener to send timestamp information to connecting Client
+def send_timestamp(timestamp, event):
+    address = ("0.0.0.0", 5005)
```

Comment:
Would we have any concerns about potential DOS attacks on this?

Reply:
Would a possible way to account for this be to create a docker network with just the `osprey` and `osprey-flask-app` containers? Or check that the address returned in `listener.accept()` is the IP address of the `osprey-flask-app` container?

Reply:
Yes, having the two containers on their own network would mitigate the risk of a DOS attack. Aside from that, it's generally a good practice to make the listening address and port to be run time parameters (probably with address defaulting to localhost/127.0.0.1), so that admins have some flexibility in how they deploy the service.

Reply:
I put the two containers in their own network.